atomic_reactor/schemas/config.json

```diff
@@ -533,7 +533,7 @@
         "type": "string"
     },
     "skip_koji_check_for_base_image": {
-        "description": "Allow base image without related koji build (insecure)",
+        "description": "Allow base image without related koji build (insecure), assuming NVR isn't found in image labels, if NVR is found in image labels koji check will be performed regardless",
```

Comment:
Could we say "... (insecure). This assumes there are no NVR labels in the base image. This option will be ignored otherwise."

Reply:
@MartinBasti  @lcarva  ?
I like mine better is more descriptive :)

Reply:
> Allow base image without a related koji build (insecure) when there are no NVR labels set on the base image. If the NVR labels are set on the base image, the check is performed regardless.

Reply:
updated :)